features/step_definitions/stepdefs.js

```
@@ -1,6 +1,7 @@
 import assert from 'node:assert';
 import { createHmac } from 'node:crypto';
 import fs from 'node:fs';
+import { readFile } from 'fs/promises';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)